segmatch_ros/package.xml

```diff
@@ -17,6 +17,7 @@
   <depend>cmake_modules</depend>
   <depend>doxygen_catkin</depend>
   <depend>eigen_catkin</depend>
+  <depend>eigen_conversions</depend>
```

Comment:
For which functionality do you exactly need this one? Would be nice to do without (the list of dependencies is already huge! ;)